User-Data Store Migration/dump-converter.py

```diff
@@ -176,17 +183,23 @@ def initialize(out):
         )
 
 
+# To write custom output after writing the `pre` variable
 def post_initialize(out):
     pass
 
 
+# To write custom output before writing the `post` variable
 def pre_finalize(out):
     if __dialect__ == "mssql" and last_header:
+        # For the last processed table, reset the standard behaviour of autoincrement
         writeline(out, ("IF OBJECTPROPERTY(OBJECT_ID('" + NAME_TEMPLATE + "'), 'TableHasIdentity') = 1 SET IDENTITY_INSERT " + NAME_TEMPLATE + " OFF;") % (last_header, last_header))
 
 
+# To write custom output after writing the `post` variable
 def finalize(out):
-    pass
+    writeline(out, "COMMIT;")

```

Comment:
this lines is new, was the missing "commit"preventing anything from working properly?

Reply:
before I wasn't deleting the existing lines. so the commit was just kept without any change. now I don't have any rule for that so the one in the input is ignored and I explicitly write it where I need it (it's more deterministic the position).

for the same reason, I removed the transformation rule for the beginning of transaction and I explicitly put it with the right syntax where I need

Reply:
understood, thank you